naiserator-dev.yaml

```diff
@@ -120,3 +120,5 @@ spec:
       value: https://oidc-ver2.difi.no/idporten-oidc-provider/
     - name: DITT_SYKEFRAVAER_FRONTEND_CLIENT_ID
       value: dev-gcp:flex:ditt-sykefravaer
+    - name: USE_KANDIDATLISTA
```

Comment:
Hva med å inkludere ordet `TOGGLE` i dette navnet så det er lett å skjønne at det er det det er? For det er vel i bunn og grunn en toggle?

Reply:
TOGGLE_KANDIDATLISTA ?